contracts/CapabilityDelegator.cdc

```diff
@@ -7,56 +7,57 @@
 /// private `Delegator` can only be borrowed from the child account when you have access to the full `ChildAccount` 
 /// resource.
 ///
-pub contract CapabilityDelegator {
+access(all) contract CapabilityDelegator {
 
     /* --- Canonical Paths --- */
     //
-    pub let StoragePath: StoragePath
-    pub let PrivatePath: PrivatePath
-    pub let PublicPath: PublicPath
+    access(all) let StoragePath: StoragePath
+    access(all) let PublicPath: PublicPath
+
+    access(all) entitlement Get
     
     /* --- Events --- */
     //
-    pub event DelegatorCreated(id: UInt64)
-    pub event DelegatorUpdated(id: UInt64, capabilityType: Type, isPublic: Bool, active: Bool)
+    access(all) event DelegatorCreated(id: UInt64)
+    access(all) event DelegatorUpdated(id: UInt64, capabilityType: Type, isPublic: Bool, active: Bool)
 
     /// Private interface for Capability retrieval
     ///
-    pub resource interface GetterPrivate {
```

Comment:
I don't Get (pun intended) why something where before all of these where restricted and now only some of them are. If that is now ok then by all means, but the semantics change.

Reply:
I am assuming you mean

```
access(all) view fun findFirstPrivateType(_ type: Type): Type?
```

Really it's just about what something can be used while moving things to entitlements. Since this one is only returning a Type object, I don't see why we would need to make it non-public personally